### PR TITLE
Individuals

### PR DESCRIPTION
**Pull request by david4096**
Adds `individuals` endpoint and the ability to filter bioSamples by `individualId`. 

https://github.com/ga4gh/server/issues/996

### CODE REVIEW

**Comment by david4096** [comment]
Most of the individual functionality is in. Known issue: IDs are not unique when biosamples and individuals have the same name.

**Comment by david4096** [comment]
Added the ability to specify an individual name at repo load time for a biosample to solve the many biosample to single individual problem.

`python repo_dev.py add-biosample ga4gh-example-data brca1 HG00097.json --individualName HG00097`

**Comment by david4096** [comment]
Known issues: 
* There is no guarantee that an individual record exists when adding and individual.
* Removing an individual will leave pointers to that individual in any biosamples linked to it.

**Comment by david4096** [comment]
How to take it for a spin: https://gist.github.com/david4096/58c68b9bedeb5e783c20

**Comment by macieksmuga** [comment]
Checked and tested!
+1 modulo minor changes to the `prepare_compliance_data.py` script to accommodate the flattening out of test data in the corresponding compliance PR https://github.com/ga4gh/compliance/pull/179

File: scripts/prepare_compliance_data.py [hunk 1]
```diff
@@ -136,12 +146,62 @@ def run(self):
                     # in place, creates a tabix index.
                     pysam.tabix_index(destFile, preset="vcf")
 
-        ontologiesDir = os.path.join(self.outputDirectory, "ontologies")
-        sequenceOntologyDir = os.path.join(ontologiesDir, "sequence_ontology")
-        os.makedirs(sequenceOntologyDir)
-        shutil.copy(os.path.join(self.inputDirectory, "sequence_ontology.txt"),
-                    os.path.join(sequenceOntologyDir, "sequence_ontology.txt"))
-        print("done converting compliance data.", file=sys.stderr)
+            # BioData
```
Comment (david4096) [inline scripts/prepare_compliance_data.py]: @macieksmuga prepare compliance data script updated!

**Pull request merged**